### ISSUE TITLE
Grab without a cart

### ISSUE THREAD

**Issue by arunavanag591** [jas497/beta_coke_fetcher#1]
So i am trying to grab the can. My baxter is on a static mounted platform. I was wondering if I can use the baxter_arm_test for grabbing my can. Also for the entire package, it looks out for cartesian_planner, a package that I could not find, and I believe it might be some thing personal. Any help with this will be appreciated!

**Comment by TuZZiX** [comment]
Yes, you do need to get cartesian planner package, all the dependency can be found here: https://github.com/cwru-robotics/cwru_baxter
They are written by Dr. Newman

I have written an moveit! planning interface, however, our Baxter have done a firmware upgrade, and moveit! will stuck in the middle when executing plan. But I think it will works on your Baxter, last year I use that for arm moving part.

**Comment by TuZZiX** [comment]
Also, you may found a newer version in the repository for the book learning ROS (not sure): https://github.com/wsnewman/learning_ros

At least it is better documented and better organized. XD
